core/src/main/java/io/seata/core/rpc/netty/MessageCodecHandler.java

```diff
@@ -50,57 +55,81 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MessageCodecHandler.class);
     private static short MAGIC = (short)0xdada;
-    private static int HEAD_LENGTH = 14;
-    private static final int FLAG_REQUEST = 0x80;
-    private static final int FLAG_ASYNC = 0x40;
-    private static final int FLAG_HEARTBEAT = 0x20;
-    private static final int FLAG_SEATA_CODEC = 0x10;
+    private static int HEAD_LENGTH = 16;
+    private static final short FLAG_REQUEST = 0x80;
+    private static final short FLAG_ASYNC = 0x40;
+    private static final short FLAG_HEARTBEAT = 0x20;
+    private static final short FLAG_SEATA_CODEC = 0x10;
     private static final int MAGIC_HALF = -38;
     private static final int NOT_FOUND_INDEX = -1;
 
+    private static Configuration configuration = ConfigurationFactory.getInstance();
+
+    private static String serialize = configuration.getConfig(ConfigurationKeys.SERIALIZE_FOR_RPC);
```

Comment:
should there be a default value that is compatible with the previous configuration file?